.github/workflows/publish-command.yml

```diff
@@ -65,9 +65,29 @@ jobs:
           aws-access-key-id: ${{ secrets.SELF_RUNNER_AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.SELF_RUNNER_AWS_SECRET_ACCESS_KEY }}
           github-token: ${{ needs.find_valid_pat.outputs.pat }}
+  preprocess-matrix:
+    needs: start-publish-image-runner
+    runs-on: ${{ needs.start-publish-image-runner.outputs.label }}
+    outputs:
+      connectorjson: ${{ steps.preprocess.outputs.connectorjson }}
+    steps:
+      - id: preprocess
+        run: |
+          start="[\""
+          replace="\",\""
+          end="\"]"
+          stripped_connector="$(echo "${{ github.event.inputs.connector }}" | tr -d ' ')"
+          middle=${stripped_connector//,/$replace}
+          full="$start$middle$end"
+          echo "::set-output name=connectorjson::$full"
```

Comment:
bash witchcraft!

Reply:
... there may be a `jq` way that might be cleaner.

Reply:
Can you add a comment about what this script is trying to do? For example:

```bash
# input: connectors/x, connectors/y, connectors/z
# output: ["connectors/x", "connectors/y", "connectors/z"]
```